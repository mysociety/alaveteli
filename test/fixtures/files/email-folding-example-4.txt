<<Freedom of Information request - Contracts or options with Kimberley
Developments or Waitrose>>

Arthur Pritchard
Property & Assets Manager
Tel: 01625 504234
Fax: 01625 504268
e-mail: [1][email address]

***********************************************************************************
The information in this Email and any attachments is personal to the
sender and the views of the author may not necessarily reflect those
of Macclesfield Borough Council. The information is strictly confidential
and is intended only for the named person or organisation to whom it is
addressed as it may contain privileged and confidential information. If
you are not the intended recipient do not copy, distribute or use this
Email, and please notify the sender. Please note that we cannot
guarantee that this message or any attachment is virus free or has not
been intercepted and amended.
***********************************************************************************

Disclaimer

--------------------------------------------------------------------------

This email message has been scanned for viruses by Mimecast.
Mimecast delivers a complete managed email solution from a single web
based platform.
For more information please visit [2]http://www.mimecast.com

--------------------------------------------------------------------------

References

Visible links
1. mailto:[email address]
2. http://www.mimecast.com/
