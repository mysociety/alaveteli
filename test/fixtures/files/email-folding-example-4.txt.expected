<<Freedom of Information request - Contracts or options with Kimberley
Developments or Waitrose>>

Arthur Pritchard
Property & Assets Manager
Tel: 01625 504234
Fax: 01625 504268
e-mail: [1][email address]
FOLDED_QUOTED_SECTION
FOLDED_QUOTED_SECTION
References

Visible links
1. mailto:[email address]
2. http://www.mimecast.com/
